src/renderers/webgl/WebGLState.js

```diff
@@ -339,6 +344,41 @@ THREE.WebGLState = function ( gl, paramThreeToGL ) {
 
 	};
 
+	this.activateTexture = function ( webglSlot ) {
```

Comment:
One last thing. Instead of `activateTexture` I would name it `activeTexture` so it matches the WebGL API.